src/main/java/com/rocketden/main/controller/v1/ProblemController.java

```diff
@@ -0,0 +1,36 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.dto.problem.Problem;
+import com.rocketden.main.dao.problem.ProblemRepository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ProblemController extends BaseRestController {
+
+	@Autowired
+	private ProblemRepository problemRepository;
+  
+  @PostMapping("/addProblem")
```

Comment:
This also may be a nit but could we change the URL paths to follow the REST patterns listed in [[1]](https://nordicapis.com/10-best-practices-for-naming-api-endpoints/) and [[2]](https://restfulapi.net/resource-naming/)? I'm not sure what this would look like exactly for our scenario, maybe something like /problems, but we can discuss this in a call.

Reply:
Absolutely, that makes sense -- changed to `"/problems"` for both the GET and POST requests.